Pod/Classes/SEGAnalytics.m

```diff
@@ -5,11 +5,11 @@
 #import "SEGAnalyticsUtils.h"
 #import "SEGAnalyticsRequest.h"
 #import "SEGAnalytics.h"
-
 #import "SEGIntegrationFactory.h"
 #import "SEGIntegration.h"
-#import <objc/runtime.h>
 #import "SEGSegmentIntegrationFactory.h"
+#import "UIViewController+SEGScreen.h"
+#import <objc/runtime.h>
```

Comment:
Ah we use it in forwarding events to methods e.g. https://github.com/segmentio/analytics-ios/blob/master/Pod/Classes/SEGAnalytics.m#L553